osu.Game/Screens/OnlinePlay/Playlists/PlaylistsRoomSettingsOverlay.cs

```diff
@@ -350,9 +350,31 @@ private void apply()
 
             private void onError(string text)
             {
-                ErrorText.Text = text;
-                ErrorText.FadeIn(50);
+                // see https://github.com/ppy/osu-web/blob/2c97aaeb64fb4ed97c747d8383a35b30f57428c7/app/Models/Multiplayer/PlaylistItem.php#L48.
+                const string not_found_prefix = "beatmaps not found:";
```

Comment:
Ideally web would provide a more structured error than this that doesn't require de-stringifying, but this works for now I guess...

Reply:
Already discussed with web team, this is best we have and likely won't change for the foreseeable future (the only more structure error web uses is `form_errors` which only gives the erroring property in addition, no json structure or anything).